lib/logger.js

```diff
@@ -3,4 +3,4 @@
  */
 const bedrock = require('bedrock');
 
-module.exports = bedrock.loggers.get('app').child('bedrock-document-server');
+module.exports = bedrock.loggers.get('app').child('document-server');
```

Comment:
I thought the decision was to leave `bedrock` in and if someone is disturbed enough by it they would write a patch in the logger to filter `bedrock` from all the things.

Reply:
Yeah, that was written earlier.  Updated.